docs/conf.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+from json import loads
+from pathlib import Path
+
+# -- Sphinx Options -----------------------------------------------------------
+
+# If your project needs a minimal Sphinx version, state it here.
+needs_sphinx = "3.0"
+
+extensions = [
+    "sphinx.ext.extlinks",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.todo",
+    "sphinxarg.ext",  # Automatic argparse command line argument documentation
```

Comment:
Comment/remove this extension if unused. Alternatively, add a `docs/requirements.txt` file.